pkg/build/controller/build/build_controller.go

```diff
@@ -1522,7 +1523,8 @@ func (bc *BuildController) createBuildSystemConfigMapSpec(build *buildv1.Build,
 	}
 	if len(bc.registryConfData) > 0 {
 		cm.Data = map[string]string{
-			buildutil.RegistryConfKey: bc.registryConfData,
+			buildutil.RegistryConfKey:    bc.registryConfData,
+			buildutil.SignaturePolicyKey: bc.signaturePolicyData,
```

Comment:
shouldn't you separately check if signaturePolicyData is empty before adding it to the CM?

Reply:
yep - missed that opening if block.